configure.ac

```
@@ -8,7 +8,7 @@ AC_PROG_LN_S
 AC_PROG_MKDIR_P
 AC_PROG_SED
 AC_ARG_WITH([pytest],[  --with-pytest=executable],[PYTEST="$withval"])
-if test -z "$PYTEST"; then
+if test ! "$PYTEST"; then
```

```suggestion
if test "x$PYTEST" = x; then
```

Not sure if we really need to care about (pre-POSIX) Bourne `sh`, but Autotools seem to target Bourne `sh`. I think it is completely fine to forget about the Bourne `sh`, but I just noticed this.

I have checked the behavior of Bourne `sh`, and it seems to fail (i.e., output an error message) with `test ! "$PYTEST"` when `$PYTEST` contains a unary operator like `-f`. On the other hand, `test -z "$PYTEST"` seems to work always as expected at least in the Bourne `sh` that I tested. I guess the Bourne `sh` way is `test "x$PYTEST" = x`.

---

Yeah, I actually thought I'd leave this one alone, but reading the POSIX spec made me think it'd work and I failed to think about the case the variable contains an operator. Thanks for pointing it out!